ledger/ledger-api-test-tool/BUILD.bazel

```diff
@@ -13,6 +25,30 @@ load(
 )
 load("@os_info//:os_info.bzl", "is_windows")
 
+daml_compile(
+    name = "PackageManagementTest",
+    main_src = "src/main/daml/PackageManagementTest.daml",
+    target = "1.dev",
```

Comment:
Why not use the latest stable daml-lf release?

Reply:
Initially I thought it wasn't particularly relevant and I got in the version I assumed was always going to be there, but my assumption is not particularly robust on a second thought. Switching to `lf_latest_version`.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/2638/commits/2227e83ce01473eba9a26d7ddc5a8b063f46fade